rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -366,11 +371,11 @@ components:
     MetadataType:
       description: The type of metadata.
       type: string
```

Comment:
❗️  Add a regex for valid URL format
✏️ Might want to update the other URIs on the transform request.